azure-functions/acbs-function/mappings/facility/helpers/get-exposure-period.test.js

```diff
@@ -0,0 +1,237 @@
+const { cloneDeep } = require('lodash');
+const getExposurePeriod = require('./get-exposure-period');
+const { PRODUCT } = require('../../../constants');
+
+describe('getExposurePeriod', () => {
+  describe('with amendment and fmr', () => {
```

Comment:
I know this is a really odd comment but can you add what fmr is in brackets here

Reply:
`facility master record`